src/components/Popups/__tests__/PopupReply-test.tsx

```diff
@@ -36,15 +36,20 @@ describe('PopupReply', () => {
     });
 
     describe('event handlers', () => {
+        const eventTypes: { [key: string]: string } = {
+            onCancel: 'click',
+            onSubmit: 'submit',
```

Comment:
Can the simulated event type be incorporated into the `test.each` table, instead?